neuronunit/tests/get_neab.py

```diff
@@ -0,0 +1,83 @@
+import os,sys
```

Comment:
@russelljjarvis 
Let's move`get_neab.py` back in the `optimization` directory.  Nothing in `tests` should really be calling it anyway.

Reply:
I think this is done but will double check.